### PR TITLE
Render text below pin

### LINKED ISSUE #54: AR: Display Title of Locations
As a user I find it confusing to see a bunch of pins without any context in the AR view. I would like to be able to differentiate between them.

### PR DESCRIPTION
**Pull request by jake-shasteen**
Partnered with @tiffeli 

In this PR we added the rendering of the pin's title below each pin.

Resolves #54 

Feedback requested on code quality and styling.

Thanks

### CODE REVIEW

**Comment by bignell** [comment]
@jake-shasteen Reviewed :+1:

**Comment by bignell** [comment]
Tapping @asu91 for second review (cc: @jake-shasteen )

File: app/lib/threejs/text.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  // var createTextModel = function( text ) {
+  //   var parameters = {
+  //     font: font,
+  //     size: 50,
+  //     height: 1,
+  //   };
+  //   var textGeometry = new THREE.TextGeometry( text, parameters );
+  //   var textMaterial = new THREE.MeshNormalMaterial();
+  //   return new THREE.Mesh( textGeometry, textMaterial );
+  // };
```
Comment (asu91) [inline app/lib/threejs/text.js]: Is there a reason all this code is commented out?

File: app/lib/threejs/text.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    var material = new THREE.MeshBasicMaterial( { map: texture, side: THREE.DoubleSide } );
+
+    return new THREE.Mesh(
+      new THREE.PlaneGeometry( 30, 30 ),
+      material
+    );
+  }
+</script>
+`;
```
Comment (asu91) [inline app/lib/threejs/text.js]: Space at the end?

**Comment by asu91** [comment]
@jake-shasteen @bignell  Reviewed. Good to go :heavy_check_mark:

**Comment by tiffeli** [comment]
@bignell i tested this too on my phone, again. It works fine.

File: app/lib/threejs/text.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+export default THREE_RENDER_TEXT = `
```
Comment (bignell) [inline app/lib/threejs/text.js]: New strict mode, you know what to do :+1:

**Pull request merged**